angular-ngrx-scss/src/app/pull-requests/pull-requests.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store } from '@ngrx/store';
+import { fetchPullRequests, selectPullRequests } from '../state/repository';
+
+@Component({
+  selector: 'app-pull-requests',
+  templateUrl: './pull-requests.component.html',
+  styleUrls: ['./pull-requests.component.scss'],
+})
+export class PullRequestsComponent implements OnInit {
+  owner!: string;
+  repoName!: string;
+  pullRequests$ = this.store.select(selectPullRequests);
+
+  constructor(private route: ActivatedRoute, private store: Store) {}
+
+  ngOnInit() {
+    this.owner = this.route.snapshot.paramMap.get('owner') as string;
```

Comment:
suggestion: Would love to see these values being pulled from the state instead of the route. We should have them there, and selectors for them could be created if the selector doesn't exist yet.

Reply:
Can leave this for a future cleanup task, as another task will be working on improving the state structure already.